### ISSUE TITLE
Getting a panic after latest release

### ISSUE THREAD

**Issue by connorbrown-8451** [databricks/setup-cli#73]
After the latest release of this action, my workflow is failing with the following error.

> panic: $HOME is not set
> 
> goroutine 1 [running]:
> github.com/databricks/cli/libs/env.UserHomeDir({0x[13](https://github.com/redacted#step:3:16)6fd18?, 0x1be9200?})
> 	github.com/databricks/cli/libs/env/context.go:82 +0x5b
> github.com/databricks/cli/cmd/labs/project.PathInLabs({0x136fd18?, 0x1be9200?}, {0x0, 0x0, 0x4?})
> 	github.com/databricks/cli/cmd/labs/project/helpers.go:[16](https://github.com/redacted#step:3:19) +0x37
> github.com/databricks/cli/cmd/labs/project.Installed({0x136fd[18](https://github.com/redacted#step:3:21), 0x1be9200})
> 	github.com/databricks/cli/cmd/labs/project/installed.go:29 +0x39
> github.com/databricks/cli/cmd/labs.New({0x136fd18, 0x1be9[20](https://github.com/redacted#step:3:23)0})
> 	github.com/databricks/cli/cmd/labs/labs.go:31 +0x471
> github.com/databricks/cli/cmd.New({0x136fd18, 0x1be9200})
> 	github.com/databricks/cli/cmd/cmd.go:74 +0x450
> main.main()
> 	github.com/databricks/cli/main.go:11 +0x[25](https://github.com/redacted#step:3:28)
> Error: Process completed with exit code 2.

Pinning this action like so fixes the issue:

```YAML      
- uses: databricks/setup-cli@7e0aeda05c4ee281e005333303a9d2e89b6ed639
  with:
     version: '0.209.0'
```

**Comment by pietern** [comment]
Thanks for reporting, @connorbrown-8451 ! Will look into this asap.

**Comment by pietern** [comment]
@connorbrown-8451 The fix is released as v0.210.1. Thanks for reporting this!
